engine/mpm_solver.py

```diff
@@ -168,8 +170,7 @@ def block_component(c):
                                                                  offset=offset)
 
             block_component(grid_m)
-            for v in grid_v.entries:
-                block_component(v)
+            block_component(grid_v)
```

Comment:
Does this change the SOA layout in the leaf blocks into AOS?

Reply:
Fixed in the latest commits.